src/parser/xmlHandler.js

```diff
@@ -437,7 +437,12 @@ class XMLHandler {
       if (value !== undefined) {
         top.object[this.options.valueKey] = value + val;
       } else {
-        top.object[this.options.valueKey] = val;
+        if (top.object["$attributes"].lang){
```

Comment:
I think there's a better way to do this generically for an object with any attributes, and updating (rather than replacing) the top level object. I'll accept this for now, but it feels dangerous ;-)